pkg/declextract/serialization.go

```diff
@@ -27,6 +27,11 @@ meta automatic
 
 type auto_todo int8
 
+type auto_union[INFERRED, RAW] [
```

Comment:
Nit: it now looks quite confusing w.r.t. our squashed arguments representation (which is also called AUTO), but I don't know a better name.

Reply:
Right. But I can't think of a better name. We use "auto" here a lot. Maybe we should rename AUTO to something else now.